src/app/app.module.ts

```diff
@@ -57,7 +57,7 @@ import {
   UserService
 }                             from 'ngx-login-client';
 import { WidgetsModule }      from 'ngx-widgets';
-import { ActionModule, ListModule, TreeListComponent, PatternFlyNgModule } from 'patternfly-ng';
+import { ActionModule, ListModule, TreeListComponent, PatternFlyNgModule, ChartModule } from 'patternfly-ng';
```

Comment:
Please list these alphabetically. Also after 2 or 3 just make it a vertical list. It should have been updated a while ago.